pontoon/settings/base.py

```diff
@@ -58,6 +58,13 @@ def path(*args):
 # tests.
 SSLIFY_DISABLE = DEBUG or os.environ.get('CI', False)
 
+# URL to the RabbitMQ server
+BROKER_URL = os.environ.get('RABBITMQ_URL', None)
```

Comment:
Yeah, but I like being explicit about it.

Except all those other places I wasn't explicit about it. Let's pretend those don't exist because I'm always consistent.